### PR TITLE
Individual switches boldfaced when 'true' in Logical Switches menu

### PR DESCRIPTION
**Pull request by alexeystn**
Implemented feature request #6812

### CODE REVIEW

File: radio/src/gui/128x64/model_logical_switches.cpp [hunk 1]
```diff
@@ -101,7 +101,7 @@ void menuModelLogicalSwitchOne(event_t event)
         lcdDrawTextAlignedLeft(y, STR_V1);
         int v1_min=0, v1_max=MIXSRC_LAST_TELEM;
         if (cstate == LS_FAMILY_BOOL || cstate == LS_FAMILY_STICKY || cstate == LS_FAMILY_EDGE) {
-          drawSwitch(CSWONE_2ND_COLUMN, y, v1_val, attr);
+          drawSwitch(CSWONE_2ND_COLUMN, y, v1_val, attr | ((getSwitch(v1_val) && v1_val) ? BOLD : 0));
```
Comment (raphaelcoeffic) [inline radio/src/gui/128x64/model_logical_switches.cpp]: the condition should be reversed: `v1_val && getSwitch(v1_val)`. This avoids evaluating `getSwitch()` for nothing.

File: radio/src/gui/128x64/model_logical_switches.cpp [hunk 2]
```diff
@@ -129,7 +129,7 @@ void menuModelLogicalSwitchOne(event_t event)
         lcdDrawTextAlignedLeft(y, STR_V2);
         int16_t v2_min = 0, v2_max = MIXSRC_LAST_TELEM;
         if (cstate == LS_FAMILY_BOOL || cstate == LS_FAMILY_STICKY) {
-          drawSwitch(CSWONE_2ND_COLUMN, y, cs->v2, attr);
+          drawSwitch(CSWONE_2ND_COLUMN, y, cs->v2, attr | ((getSwitch(cs->v2) && cs->v2) ? BOLD : 0));
```
Comment (raphaelcoeffic) [inline radio/src/gui/128x64/model_logical_switches.cpp]: same everywhere else

File: radio/src/gui/128x64/model_logical_switches.cpp [hunk 3]
```diff
@@ -185,7 +185,7 @@ void menuModelLogicalSwitchOne(event_t event)
 
       case LS_FIELD_ANDSW:
         lcdDrawTextAlignedLeft(y, STR_AND_SWITCH);
-        drawSwitch(CSWONE_2ND_COLUMN, y, cs->andsw, attr);
+        drawSwitch(CSWONE_2ND_COLUMN, y, cs->andsw, attr | ((getSwitch(cs->andsw) && cs->andsw) ? BOLD : 0));
```
Comment (raphaelcoeffic) [inline radio/src/gui/128x64/model_logical_switches.cpp]: same everywhere else

File: radio/src/gui/128x64/model_logical_switches.cpp [hunk 4]
```diff
@@ -294,15 +294,15 @@ void menuModelLogicalSwitches(event_t event)
       uint8_t cstate = lswFamily(cs->func);
 
       if (cstate == LS_FAMILY_BOOL || cstate == LS_FAMILY_STICKY) {
-        drawSwitch(CSW_2ND_COLUMN, y, cs->v1, 0);
-        drawSwitch(CSW_3RD_COLUMN, y, cs->v2, 0);
+        drawSwitch(CSW_2ND_COLUMN, y, cs->v1, (getSwitch(cs->v1) && cs->v1) ? BOLD : 0);
```
Comment (raphaelcoeffic) [inline radio/src/gui/128x64/model_logical_switches.cpp]: same everywhere else

**Review by raphaelcoeffic — CHANGES_REQUESTED** [review]

**Comment by bsongis** [comment]
To avoid duplicated code I think you could add a parameter to `drawSwitch`? Or introduce a new function?

**Comment by bsongis** [comment]
Done in #7000

**Pull request closed without merging**